src/xclim/ensembles/_partitioning.py

```diff
@@ -13,6 +13,8 @@
 import pandas as pd
 import xarray as xr
 
+from xclim.sdba import loess
```

Comment:
hmm, can we do without loess ? Technically, importing anything from `sdba` outside `xclim.sdba` is forbidden.

I would argue that xclim is not the place where to perform smoothing and one solution would to expect `sm` to be computed outside the function.

Another solution, it to change the LOESS to a rolling mean with some weighting ? As we discussed before, one good reason to use a LOESS vanishes if we have an equally-spaced timeseries (which you actually constrain here, line 361). I would suggest using a rolling mean with some weighting window generated with [scipy](https://docs.scipy.org/doc/scipy/reference/generated/scipy.signal.windows.get_window.html).

Reply:
I wanted to use loess for the paper, but it turned out to not work super well with some indicators, so I went back to poly. I will remove it from the function. Computing sm outside the function is already an option.